### ISSUE TITLE
Enable notifications

### ISSUE THREAD

**Issue by Putsomethininthemicrowave** [gitnew2018/My-OpenPeriscope#6]
"Enable Notifications" option was working in an ancient version of OP (Windows) last year. 
I remember it giving me proper Windows notifications for everything I selected.
Hasn't been working ever since in the standalone application.
Are they working for you?

**Comment by gitnew2018** [comment]
No they don't work for me, even in open periscope.

**Comment by gitnew2018** [comment]
I'm not sure yet but i think it's a problem with the system and not he app.

**Comment by ddpk** [comment]
There was this registry edit you could do to fix it but after I tried it again, it didn't work. After windows update. What does "After 1 release" mean by the way? Are there features not available right now?

**Comment by ddpk** [comment]
Ah, so I see a lot of the features I really want aren't in the latest release. Any chance of a new release soon? Or if there is a guide to updating it myself (standalone version), although I don't know much so I wouldn't understand simple directions.. haha thank you! I may have feature ideas, but i'd love to first see the latest version in action :)

**Comment by gitnew2018** [comment]
@ddpk you can build it yourself [wiki](https://github.com/gitnew2018/My-OpenPeriscope/wiki), if something is not clear ask.
I also want to release ver. 0.2 soon but you know what soon means...

**Comment by ddpk** [comment]
@gitnew2018 I tried building it, but must have gotten lost somewhere with npm install because it never gave me the ["result"](https://user-images.githubusercontent.com/37026885/42737947-e638bf88-887b-11e8-8c18-17edb78b99cd.png) . Then I tried Option 2 for merging to an executable. I had no such things that "npm added" in the mix and the exe resulted into nothing but a nw:blank. I'm very confused why downloading the old exe and all the (new?) edited source files and putting them all into one folder, then launching the exe doesn't work. But i'm so new to all of this and wish I knew better lol, thanks for the help, hopefully I can become a bit better at this by the end of today :)
